modules/nf-core/wittyer/main.nf

```diff
@@ -0,0 +1,94 @@
+//               e.g. it's ok to have a single module for bwa to output BAM instead of SAM:
+//                 bwa mem | samtools view -B -T ref.fasta
+// TODO nf-core: Optional inputs are not currently supported by Nextflow. However, using an empty
+//               list (`[]`) instead of a file can be used to work around this issue.
+
+process WITTYER {
+    tag "$meta.id"
+    label 'process_single'
+
+    // TODO nf-core: List required Conda package(s).
+    //               Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    //               For Conda, the build (i.e. "h9402c20_2") must be EXCLUDED to support installation on different operating systems.
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
+    // conda "${moduleDir}/environment.yml"
+    container "nf-core/wittier:0.3.3"
+    // "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    //    'https://depot.galaxyproject.org/singularity/YOUR-TOOL-HERE':
+    //    'biocontainers/YOUR-TOOL-HERE' }"
+
+    input:
+    // TODO nf-core: Where applicable all sample-specific information e.g. "id", "single_end", "read_group"
+    //               MUST be provided as an input via a Groovy Map called "meta".
+    //               This information may not be required in some instances e.g. indexing reference genome files:
+    //               https://github.com/nf-core/modules/blob/master/modules/nf-core/bwa/index/main.nf
+    // TODO nf-core: Where applicable please provide/convert compressed files as input/output
+    //               e.g. "*.fastq.gz" and NOT "*.fastq", "*.bam" and NOT "*.sam" etc.
+    tuple val(meta), path(query_vcf), path(query_vcf_tbi), path(truth_vcf), path(truth_vcf_tbi), path(bed)
+
+    output:
+    tuple val(meta),    path("*.Stats.json")         , emit: report
+    tuple val(meta),    path("*eval.vcf.gz")         , emit: bench_vcf
+    tuple val(meta),    path("*eval.vcf.gz.tbi")     , emit: bench_vcf_tbi
+    path  "versions.yml" 
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        error "WHITTYER module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+    def args  = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def regions = bed ? "--includeBed=$bed" : ""
+    """
+    mkdir bench
+    wittyer \\
+        --truthVcf=${truth_vcf} \\
+        --inputVcf=${query_vcf} \\
+        --outputDirectory=bench \\
+        ${regions} \\
+        ${args}
+
+    mv bench/Wittyer.Stats.json ${prefix}.Stats.json
+    mv bench/*.vcf.gz ${prefix}.eval.vcf.gz
+    mv bench/*.vcf.gz.tbi ${prefix}.eval.vcf.gz.tbi
```

Comment:
Could you remove any custom extensions that are not file extensions? This should be done to make all filenames as customizable as possible :)

Reply:
I think `eval` was added to show that this vcf is the result of the benchmarking comparison. Should that be captured via `prefix` then?

Reply:
Yes that should be added via the `prefix`. It could also be a good idea to add a simple check to see if the output filename isn't the same as the input files (otherwise Nextflow will give some weird errors) => See https://nf-co.re/docs/contributing/modules#naming-conventions part 6 for an example

Reply:
but the name is generated through wittyer, I think it makes sense to keep original extensions.

Reply:
_eval_ is already exist in original output, only prefix is being added here to the name. I would prefer it in this way actually.

Reply:
I understand, but you lose the flexibility to call the file whatever you want. You can still add `eval` to the output name yourself if you want that :) (I'm just reciting what's in the guidelines, so don't shoot the messenger :wink:)

Reply:
But I was not adding the extension by myself :D it is the standard output by the tool.

Reply:
This type of names are important for traceability with standard outputs of the tool. So I would really prefer to keep is as it is.

Reply:
But the `eval` is only added when you move the file from the output directory?

Reply:
no, I am copying the result here. eval is in standard output.